### PR TITLE
KOGITO-2639 Follow up upgrade dependencies aliging with Quarkus 1.6.0

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-runtimes/pull/607
- https://github.com/kiegroup/kogito-examples/pull/290
- https://github.com/kiegroup/kogito-apps/pull/305

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by evacchi — APPROVED** [review]
so my understanding is that is to keep the subproject aligned to 1.4.1 right?

**Comment by cristianonicolai** [comment]
@evacchi yep, upgrading the services is another story :)

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: pom.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
   <properties>
     <version.io.quarkus>1.4.1.Final</version.io.quarkus>
+    <version.io.smallrye.reactive>2.0.2</version.io.smallrye.reactive>
```
Comment (tiagodolphine) [inline pom.xml:20]: There were errors using the `2.1.1`?

Comment (cristianonicolai) [reply pom.xml:20]: @tiagodolphine yep, for both Job Service and Data Index. Could you open a JIRA to look into that?

Comment (tiagodolphine) [reply pom.xml:20]: Ok, I'll do it.

Comment (tiagodolphine) [reply pom.xml:20]: https://issues.redhat.com/browse/KOGITO-2647

**Pull request merged**